SUMMARY.md

```diff
@@ -104,55 +76,3 @@
 
 ## Legal
 * [Terms and Conditions](book/legal/terms.md)
-
----
-<!---
-
-### Getting Started with Wallaroo 2
-
-
-* [Wallaroo Concepts](wallaroo-concepts.md)
-* [Installing Wallaroo](installing-wallaroo.md)
-* [Hello, Wallaroo!](hello-wallaroo.md)
-* [Starting a Cluster](starting-a-cluster.md)
-* [Building an Application](building-an-application.md)
-* [Exploring Core Features](exploring-core-features.md)
-* [Test Page](test-page.md)
-
-### Develop
-* [Wallaroo API](wallaroo-api.md)
-* [Topologies](topologies.md)
-* [Language Bindings](Language Bindings/readme.md)
-  * [Pony](Language Bindings/pony.md)
-  * [C++](cpp.md)
-
-### Deploy
-* [Recommended Production Settings](recommended-production-settings.md)
-* [Manual Deployment](manual-deployment.md)
-* [Cloud Deployment](cloud-deployment.md)
-* [Start a Node](start-a-node.md)
-* [Stop a Node](stop-a-node.md)
-
-### Manage
-* [Admin UI](admin-ui.md)
-* [Troubleshoot](troubleshoot.md)
-
-### Learn How it Works
-* [Frequently Asked Questions](FAQ.md)
-* [Wallaroo in Comparison](wallaroo-comparison.md)
-* [Wallaroo Architecture](wallaroo-architecture.md)
-* [Wallaroo Features](wallaroo-features.md)
-* [Demo - Market Spread Application](demo-market-spread-application.md)
-
-### Misc
-* [Distributed Computing Resources](distributed-computing-resources.md)
-
-
-### Contribute
-* [Contribute to Wallaroo](contribute-to-wallaroo.md)
-* [Improve the Docs](improve-the-docs.md)
-
-### Release Notes
-* [Wallaroo Roadmap](roadmap.md)
-* [v1.0-201611101](v1.0-201611101.md)
--->
```

Comment:
This 2nd removal is unrelated to removing C++ stuff and should be in a different commit.

Reply:
Moved the unrelated SUMMARY.md changes to https://github.com/WallarooLabs/wallaroo/pull/2349